test/runtests.jl

```diff
@@ -25,3 +25,22 @@ end
     @test dnorm < 2.0
 end
 
+@testset "Swapping Ig1 with Ig2 doesn't change anything." begin
+    rr = TAMode.param(params)
+    rrSwap = TAMode.swapIgs(rr)
+
+    data = TAMode.runTAM(tps, rr, 10)
+    dataSwap = TAMode.runTAM(tps, rrSwap, 10)
+
+    for ii in 1:size(data.pY)
+        @test data.pY .≈ dataSwap.pY
```

Comment:
@sumedha-k this is much easier in Julia. See the line here and please fix the others accordingly.

I'll setup the vectors for calculating these quantities, since we don't have the `get` methods.

Reply:
P.S. Don't think there are test messages in julia, so you can write that line the same as the others.